admin_guide/sdn_troubleshooting.adoc

```diff
@@ -630,6 +630,23 @@ provide the output from the debugging script)
 * Kube - declare as a LoadBalancer and something _you_ have to write does the rest
 * OS/AE - Both use the router
 
+[[tls-handshake-timeout]]
+=== TLS Handshake Timeout
+
+When a pod fails to deploy, check its docker log for a TLS handshake timeout:
+
+----
+$ docker log <container_id>
+...
+[...] couldn't get deployment [...] TLS handshake timeout
```

Comment:
What are the [...]'s supposed to be? Pods name variables? I realise this is example readout, but would they be better as <pod_name> or a made up name like pod1?